CHANGELOG.md

```diff
@@ -41,6 +41,12 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Deprecated
 
 ### Removed
+- **Character**:
+  - `isNumber` is replaced by `isNumber` from Swift's Standard Library. [#689](https://github.com/SwifterSwift/SwifterSwift/pull/689) by [RomanPodymov](https://github.com/RomanPodymov).
```

Comment:
Can you just put all the same line like: 'isNumber', 'isLetter', ... extensions were removed because they are now on the Swift's Standard Library ?

Reply:
Hello @LucianoPAlmeida 
OK, I can do it, but as I can see you always put each added/removed/modified function to a new line. That's why I did it this way.